tests/nonbonded/test_nonbonded_jit.py

```diff
@@ -12,7 +12,7 @@ def test_nonbonded_reference_jittable(rng: np.random.Generator):
     U_ref = Nonbonded(
         N,
         exclusion_idxs=jnp.zeros((0,)),
-        scale_factors=jnp.zeros((0, 1)),
+        scale_factors=jnp.zeros((0, 2)),
```

Comment:
Why did this change?

Reply:
Scale factors should have a shape of (N, 2), otherwise the test fails at https://github.com/proteneer/timemachine/blob/726028fb0446f6f90c3bae93528ead31e860f74a/timemachine/potentials/nonbonded.py#L127